code/datums/uplink_item.dm

```diff
@@ -1060,30 +1060,31 @@ var/list/uplink_items = list()
 
 /datum/uplink_item/syndie_coop
 	category = "Cooperative Cell"
+	jobs_excluded = list("Nuclear Operative", "challenger")
```

Comment:
replace `"challenger"` with `CHALLENGER` (no quotes)

Reply:
done, compiled, and tested